assets/scripts/streets/Geolocation.jsx

```diff
@@ -0,0 +1,156 @@
+/* global fetch */
+import React from 'react'
+import { connect } from 'react-redux'
+import { setSettings } from '../users/settings'
```

Comment:
It's a common practice to sort imports so that libraries are imported first, local scripts are imported later. Let's move this line down below the library imports.